team/bundles/org.eclipse.jsch.ui/src/org/eclipse/jsch/internal/ui/preference/PreferencePage.java

```diff
@@ -485,6 +506,8 @@ public void widgetSelected(SelectionEvent e){
 					if(e.widget==keyGenerateDSA){
 						type=KeyPair.DSA;
 						_type=IConstants.DSA;
+						if (keyGenerateSize.getSelection()>DSA_KEY_SIZE)
+							keyGenerateSize.setSelection(DSA_KEY_SIZE);
```

Comment:
Please always use braces:
```suggestion
						if (keyGenerateSize.getSelection()>DSA_KEY_SIZE) {
							keyGenerateSize.setSelection(DSA_KEY_SIZE);
						}
```